core/domain/exp_jobs_one_off.py

```diff
@@ -30,10 +31,13 @@
 import feconf
 import utils
 
-(base_models, exp_models,) = models.Registry.import_models([
-    models.NAMES.base_model, models.NAMES.exploration])
+(file_models, base_models, exp_models,) = models.Registry.import_models([
+    models.NAMES.file, models.NAMES.base_model, models.NAMES.exploration])
 
 _COMMIT_TYPE_REVERT = 'revert'
+# pylint: disable=line-too-long
```

Comment:
Nope, find a way to do this without the pragma. You can break after "[".

/cc @apb7 -- maybe we should find a way to include you on the review when devs add `pylint: disable...`. Is this something oppiabot can handle? /cc @kevinlee12

Reply:
I'm going to err on the side of not really because of the limited resources the bot operates under, but I'll talk with @apb7 offline about this.

Reply:
Thanks! I don't think this happens too often, actually -- but yeah, I guess this requires looking at the contents of every PR, so not sure how bad that would be. Thanks for checking!

Reply:
Discussed this with @kevinlee12. Looking at some possible way so as to not increase the server load and response time. Thanks!

Reply:
Done! Removed!